WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift

```diff
@@ -0,0 +1,94 @@
+        landscapeTopConstraint =  cityTextField.topAnchor.constraint(equalTo: view.topAnchor, constant: 60.0)
+        portraitTopConstraint =  cityTextField.topAnchor.constraint(equalTo: view.topAnchor, constant: 260.0)
+        view.backgroundColor = .white
+
+        cityTextField.delegate = self
+        cityTextField.placeholder = "Write city name here"
+        cityTextField.borderStyle = .roundedRect
+        cityTextField.backgroundColor = .black
+        cityTextField.textColor = .white
+
+        cityTextField.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(cityTextField)
+
+        NSLayoutConstraint.activate([
+            cityTextField.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            cityTextField.widthAnchor.constraint(equalToConstant: 220.0),
+            cityTextField.heightAnchor.constraint(equalToConstant: 40.0)
+        ])
+        updateConstraintForCurrentOrientation()
+
+        temperatureLabel.text = "Temperature _ ℃"
+        temperatureLabel.textAlignment = .center
+        temperatureLabel.textColor = .black
+
+        temperatureLabel.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(temperatureLabel)
+        NSLayoutConstraint.activate([
+            temperatureLabel.topAnchor.constraint(equalTo: cityTextField.topAnchor, constant: 60.0),
+            temperatureLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            temperatureLabel.widthAnchor.constraint(equalTo: cityTextField.widthAnchor),
+            temperatureLabel.heightAnchor.constraint(equalTo: cityTextField.heightAnchor)
+        ])
+
+        humidityLabel.text = "Humidity _ %"
+        humidityLabel.textAlignment = .center
+        humidityLabel.textColor = .black
+
+        humidityLabel.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(humidityLabel)
+        NSLayoutConstraint.activate([
+            humidityLabel.topAnchor.constraint(equalTo: temperatureLabel.topAnchor, constant: 40.0),
+            humidityLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            humidityLabel.widthAnchor.constraint(equalTo: cityTextField.widthAnchor),
+            humidityLabel.heightAnchor.constraint(equalTo: cityTextField.heightAnchor)
+        ])
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+    }
+
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        updateConstraintForCurrentOrientation()
+    }
+
+    private func updateConstraintForCurrentOrientation() {
+        if UIDevice.current.orientation.isLandscape {
+            portraitTopConstraint.isActive = false
+            landscapeTopConstraint.isActive = true
+        } else {
+            landscapeTopConstraint.isActive = false
+            portraitTopConstraint.isActive = true
+        }
```

Comment:
```suggestion
        let isLandscape = UIDevice.current.orientation.isLandscape
        portraitTopConstraint.isActive = !isLandscape
        landscapeTopConstraint.isActive = isLandscape
```

Reply:
resolved in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/17/commits/f6bd9482b6d7960d05e03b5d079bf5e2a28fe439)